contributor/05-benefits-of-contribution-article.asciidoc

```diff
@@ -14,72 +14,72 @@ their purview?
 
 When deciding whether to create and contribute a fix instead of maintaining your
 own workaround, think about the benefit that the contribution will bring to
-your own changes.  Instead of working in isolation, those working on the upstream
+the quality of your own changes. Instead of working in isolation, those in the upstream
 project will be able to not only review but also improve your solution. You get
 support and mentoring which greatly speeds up your own development effort.
 
 Spending more time with others means that over time you will learn how the team
 works, how it organises itself, which tooling it uses in which way to build
 their project. Oftentimes your own projects will benefit from that experience:
-instead of only reading about some new library or build system you'll be able to
-gain practical experience with it before going ahead and introducing that in
+instead of only reading about some new library or build system, you'll be able to
+gain practical experience with it before going ahead and introducing it in
 your own projects. Working on more than one core project means that you will be
 exposed to a larger ecosystem from which to draw best practices and solutions to
 challenges.
 
 A nice side effect of being able to spend some of your time in other teams is
 that your reputation and impact expand the boundaries of your own team. So in
-addition to yourself learning and growing from others you get to influence
-projects.  You influence directly via your own contributions but also by
-sharing your experience and knowledge on project tooling and setup.  This sharing might
+addition to learning from others and growing yourself, you get to influence
+projects.  You influence directly via your own contributions and by
+sharing your experience and knowledge on project tooling and setup. This sharing might
 help the upstream project improve and accelerate development cycles.
 
 Aside from all of these objective criteria there is a component that is very
-hard to meassure but has been reported both in InnerSource and in Open Source
+hard to meassure, but has been reported both in InnerSource and in Open Source
 projects alike: people participate because they find work on those projects
-personally fullfilling and fun. Most likely, the aspect of being in a position
+personally fulfilling and fun. Most likely, the aspect of being in a position
 to truly self select tasks to work on does play an important role.
 This self selection typically also leads to host projects being very welcoming
-and supportive as those are key to keeping motivation in contributors up.
+and supportive in their effort to attract motivated contributors.
 
 === Team level motivation
 
-Remember that annoying bug that's suddenly fixed upstream? Why should your
-team spend the extra effort to contribute that fix to the upstream project?
+Remember that annoying bug that's been suddenly fixed upstream? Why should your
```

Comment:
Just from reading this now I wonder if we should add a word about the origin of this fix, i.e. that the host team actively created it and is maintaining it. Opinions here?

Reply:
I think "suddenly fixed upstream" does imply that "your team" had nothing to do with the fix other than enjoy the result, and I would not want to imply that it was done by the host team (what if there were other contributors?)
If I were to make one change here, I would change to "FINALLY fixed upstream" to indicate that "your team" was indeed suffering until that happened.
Thoughts?

Reply:
SGTM

Reply:
Et fecit